pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools >= 30.3.0", "wheel"]
 
 
 [tool.black]
-target-version = ["py37", "py38"]
+target-version = ["py37", "py38", "py39", "py310"]
```

Comment:
I cherry-picked the Python 3.10 commit to `main`, so this will disappear if you rebase.